estate/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    'name': "My Awesome module Roole",
+    'version': '1.0',
+    'depends': ['base'],
+    'author': "roole",
+    'application': True,
+    'installable': True,
+    'category': 'Tutorials',
+    'description': """
+    Description text
+    """,
+    'license': 'GPL-3',
```

Comment:
```suggestion
    'license': 'LGPL-3',
```
We use LGPL-3 as Open-Source/Community license by default. (And `OEEL-1` for Enterprise modules.)